src/components/SearchModal/CurrencySearch.tsx

```diff
@@ -104,20 +104,24 @@ export function CurrencySearch({
   const filteredSortedTokens = useSortTokensByQuery(debouncedQuery, sortedTokens)
 
   const native = useNativeCurrency()
+  const wrapped = native.wrapped
 
-  const filteredSortedTokensWithETH: Currency[] = useMemo(() => {
-    // Use Celo ERC20 Implementation and exclude the native asset
-    if (!native) {
-      return filteredSortedTokens
-    }
-
+  const displayCurrencies: Currency[] = useMemo(() => {
     const s = debouncedQuery.toLowerCase().trim()
-    if (native.symbol?.toLowerCase()?.indexOf(s) !== -1) {
+    let tokens: Currency[] = filteredSortedTokens
+    if (wrapped.symbol?.toLowerCase()?.indexOf(s) !== -1) {
+      // Always bump the wrapped token to the top of the list.
```

Comment:
Simplify by moving this out of the condition. Instead of L111, you can have

    const tokens = filteredSortedTokens.filter((t) => {
      if (t.equals(wrapped) || disableNonToken && t.isNative) return false
      return true
    })
    // This would be an alternative to the check mentioned for L107.
    const natives = native.equals(wrapped) ? [native] : [native, wrapped]
    return [...natives, ...tokens]